lint.ignore

```diff
@@ -693,3 +693,8 @@ SET TIMEOUT: editing/crashtests/outdent-across-svg-boundary.html
 SET TIMEOUT: editing/crashtests/textarea-will-be-blurred-by-focus-event-listener.html
 SET TIMEOUT: mathml/crashtests/mozilla/*
 PARSE-FAILED: mathml/crashtests/mozilla/289180-1.xml
+
+CONSOLE: speculation-rules/prefetch/resources/utils.sub.js
+CONSOLE: speculation-rules/prefetch/resources/executor.sub.html
+CONSOLE: speculation-rules/prefetch/no-vary-search/executor-non-eager-nvs-with-hint.html
+CONSOLE: resources/testdriver.js
```

Comment:
I don't see a reason to insert these, please explain?

Reply:
Looks like they were removed from the final patch.